pkg/kubelet/cm/deviceplugin/types.go

```diff
@@ -38,13 +36,23 @@ type Manager interface {
 
 	// Allocate takes resourceName and list of device Ids, and calls the
 	// gRPC Allocate on the device plugin matching the resourceName.
```

Comment:
These two lines are incorrect now.

Reply:
You are right. Tried to merge the two comments blobs from before.
I will rewrite this one.